### PR TITLE
HW1 jp4013

### PR DESCRIPTION
**Pull request by jaesungpark42**
New to github, but I think this is it.

### CODE REVIEW

File: jp4013.md [hunk 1]
```diff
@@ -13,7 +13,17 @@ GitHub README: [link](https://github.com/jaesungpark42/jaesungpark42/blob/main/R
 -Job Application Autofiller: There are many, but each lack flexibility on different variety of job application versions, also the only open source one is really small and lacking https://github.com/lovincyrus/job-autofiller  
 -Saving multiple pages of websites to a pdf. 
 
+**LINK**
+projects/python/chatGPTCoverLetterGenerator2.md
```
Comment (timkpaine) [inline jp4013.md:17]: use a markdown link

File: jp4013.md [hunk 2]
```diff
@@ -13,7 +13,17 @@ GitHub README: [link](https://github.com/jaesungpark42/jaesungpark42/blob/main/R
...
+
+https://github.com/jaesungpark42/project-proposals-s2023/blob/67f8d72b6de5506c257d7be5f9b324767ee7d658/projects/python/chatGPTCoverLetterGenerator2.md
+
 ## Project Contributions
 -Could be the job autofiller, but it is really small, I don't know if it would be an average experience of contributing to open source that this project intends to encourage.
 
 Really open to suggestions for contributions. I don’t have many tools/libraries that I use regularly that I have a problem with.
+
+**LINK**
+projects/javascript/jobAppAutofiller
```
Comment (timkpaine) [inline jp4013.md:27]: use a markdown link

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: projects/javascript/jobAppAutofiller [hunk 3]
```diff
@@ -0,0 +1,5 @@
+https://github.com/lovincyrus/job-autofiller
```
Comment (timkpaine) [inline projects/javascript/jobAppAutofiller:1]: should be a markdown file

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: projects/javascript/jobAppAutofiller (hunk 3 above)
Comment (jaesungpark42) [reply projects/javascript/jobAppAutofiller:1]: There was no issue link, it was what I encountered when trying to download the application.

**Review by jaesungpark42 — COMMENTED** [review]
For the contribution project, there was no issue link. I discovered the error (manifest v2 is outdated, switch to manifest v3) while trying to use the file.

**Comment by timkpaine** [comment]
lgtm

**Pull request closed without merging**